ui/dashboard.ts

```diff
@@ -247,7 +247,7 @@ ipcRenderer.on('failed', () => {
 
 export function init_dashboard() {
     // add callbacks for dashboard to the monitor
-    monitor_add_callback('query_exchange_balances', (result: ExchangeBalanceResult) => {
+    monitor_add_callback('query_exchange_balances_async', (result: ExchangeBalanceResult) => {
```

Comment:
Oh yeah I forgot about this sorry.

Reply:
no worries, I am still testing. Will take some time till I am sure it works fine.

Reply:
It makes sense it is a huge change. I am almost certain I broke something somewhere.

Reply:
yep deleting editing otc trades also does not work. Will try to see why.

Reply:
I am getting:

```
otctrades.ts:55 Uncaught TypeError: Cannot read property 'id' of undefined
    at delete_otc_trade (otctrades.ts:55)
    at jQuery.fn.init.callback (utils.ts:210)
    at HTMLLIElement.itemClick (jquery.contextMenu.min.js:1)
    at HTMLDocument.dispatch (/home/lefteris/w/rotkehlchen/node_modules/jquery/dist/jquery.js:5183)
    at HTMLDocument.elemData.handle (/home/lefteris/w/rotkehlchen/node_modules/jquery/dist/jquery.js:4991)
```

In both edit and delete seems that `const data = row.data() as OtcTrade;` returns undefined.

Why would that be happening?

Reply:
I am not sure, let me have a look. Maybe it binds another object.